.env

```diff
@@ -1,5 +1,3 @@
 # version for opensearch & opensearch-dashboards docker image
-VERSION=2.6.0
-# auth details - change according to your system
-USER=admin
-PASSWORD=admin
+VERSION=2.9.0
```

Comment:
Will this cause an issue with building on 'main', b/c the OS 3.0 docker image published as "latest" (there is no docker-hub published tag for 3.0)

Reply:
this is subject to the one using this docker - he needs to change this for his own needs - Its also stated inside the docker usage readme